wordpress-mobile/WordPress-iOS
# Update media editor version.

Closes #4003

How to test:

Get a device with a big media library: more than 5000 photos
Create or Edit a post
Click on the media icon
Check if the app doesn't crash and/or the total memory being used is not spiking. You can use the the debug session tab on Xcode.

Needs Review: @aerych
